test-utilities/modules-mock.js

```diff
@@ -1,48 +1,79 @@
-module.exports = {
-    wrappedRedisClient: {
-        hmset: function(key, hset, done){},
-        quit: function(done){},
-        hgetall: function(key, done){},
-        del: function(key, done){},
-        on: function(done){},
-        set: function(key, hset, done){},
-        sadd: function(key, hset, done){},
-        smembers: function(key, done){},
-        srem: function(key, done){}
-    },
-    redisClient: {
-        getShopAsync: async function () {},
-        getAllShopDomainsByDopplerDataAsync: async function () {},
-        getAllShopDomainsByDopplerApiKeyAsync: async function () {},
-        getAllShopDomainsByDopplerAccountNameAsync: async function () {},
-        storeShopAsync: async function () {},
-        removeShopAsync: async function () {},
-        quitAsync: async function(){}
+const sinon = require('sinon');
+
+const factories = {
+  wrappedRedisClient: () => ({
+    hmset: () => {},
+    quit: () => {},
+    hgetall: () => {},
+    del: () => {},
+    on: () => {},
+    set: () => {},
+    sadd: () => {},
+    smembers: () => {},
+    srem: () => {},
+  }),
+  redisClient: () => ({
+    getShopAsync: function() {},
+    getAllShopDomainsByDopplerDataAsync: () => {},
+    getAllShopDomainsByDopplerApiKeyAsync: () => {},
+    getAllShopDomainsByDopplerAccountNameAsync: () => {},
+    storeShopAsync: () => {},
+    removeShopAsync: () => {},
+    quitAsync: () => {},
+  }),
+  dopplerClient: () => ({
+    AreCredentialsValidAsync: () => {},
```

Comment:
A very small comment I think here `AreCredentialsValidAsync` should be like the rest `areCredentialsValidAsync` right?

Reply:
You are right, but it is out of the scope of this PR, for that reason I preferred do not include that change.